cv/lane_detection/launch/lane_detection.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+    <node
+        name="cv_model"
+        pkg="lane_detection"
+        type="lane_detection_inference_node"
+        output="screen"
+    >
+        <param name="instance_name" value="unet_inference"/>
+        <param name="model_path" value="/tmp/unet_with_sigmoid.onnx"/>
```

Comment:
@spencerteetaert 

Where to put model path?

Reply:
I hadn't figured out relative paths in ros environments so had it as a global on my system. Ideally if we can have the models in a folder like /lane_detection/models/model.onnx that would be prefered. Something that's easy to switch out. We can set a global path on the jetson/laptop systems as well, but it may be okay to just leave it as a placeholder on the git cause we aren't wanting to upload the .onnx model

Reply:
We'll deal w/ it later then